src/Miunie.Configuration/ConfigManager.cs

```diff
@@ -1,10 +1,50 @@
 using System.Configuration;
+using System;
+using System.Linq;
 
 namespace Miunie.Configuration
 {
     public class ConfigManager : IConfiguration
     {
-        public string GetValueFor(string key) => ConfigurationManager.AppSettings[key];
+        public string GetValueFor(string key)
+        {
+            var value = ConfigurationManager.AppSettings[key];
+
+            if (value is null)
+            {
+                var editor = new ConfigurationFileEditor();
+
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Configuration Error:");
```

Comment:
Shouldn't we filter out this hard-coding?

Reply:
This is an edge case so not in this case.

Reply:
Okay, was just wondering.